tests/test_multiband.py

```diff
@@ -0,0 +1,792 @@
+# This file is part of afw.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (http://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""
+Tests for Multiband objects
+"""
+import unittest
+import operator
+
+import numpy as np
+
+import lsst.utils
+import lsst.utils.tests
+from lsst.geom import Point2I, Box2I, Extent2I
+from lsst.afw.geom import SpanSet, Stencil
+from lsst.afw.detection import GaussianPsf, Footprint, makeHeavyFootprint, MultibandFootprint, HeavyFootprintF
+from lsst.afw.image import ImageF, Mask, MaskPixel, MaskedImage, ExposureF, MaskedImageF
+from lsst.afw.image.multiband import MultibandPixel, MultibandImage, MultibandMask, MultibandMaskedImage
+from lsst.afw.image.multiband import MaskedPixel, MultibandMaskedPixel, MultibandExposure
+
+
+def _testImageFilterSlicing(cls, mImage, singleType, bbox, value):
```

Comment:
I thought that the typical syntax for a function in python that takes a class as the first argument is to use `cls` as opposed to `self`. I did add the docstrings as requested.

Reply:
Ah, I was thinking that you were passing in the TestCase instance as the first argument. The fact that you aren't also explains the use of vanilla asserts.  So this is indeed okay, though it might be even better to make these TestCase methods so you can use asserts that produce better error messages.

Reply:
Sorry, I explained this wrong and now realize why I need to rename them. These are instances of TestCases, so I should probably rename `cls` to `testCase`.